native/packages/react-native-bpk-component-alert/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "react-native-bpk-component-alert",
+  "version": "0.0.01",
```

Comment:
This should be `0.0.0`. When we do a release, Lerna increments it automatically.